core/js/src/main/scala/dtc/js/JSDate.scala

```diff
@@ -119,3 +128,14 @@ object JSDate {
     new JSDate(jsDate)
   }
 }
+
+private[js] object JSDateCompatibility {
+
+  /**
+    * Since Scala.js 1, JSDate works with Doubles instead io Ints.
+    * See https://github.com/scala-js/scala-js/issues/2751.
+    * This is just to reduce amount of back and forth conversions between Int and Double.
+    */
+  implicit def doubleToInt(x: Double): Int = x.toInt
```

Comment:
kinda ugly, but WYGD

Reply:
Yeah, otherwise you have to have 2 versions if JSDate to 0.6 and for 1.0. Which is also... meh..

Reply:
fair